charts/main/templates/deployment.yaml

```
@@ -5,6 +5,10 @@ metadata:
   labels:
     {{- include "main.labels" . | nindent 4 }}
 spec:
+  {{- if .Values.strategy }}
+  strategy:
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)